packages/story-editor/src/components/library/panes/shopping/shoppingPane.js

```diff
@@ -269,20 +292,31 @@ function ShoppingPane(props) {
             <ProductSort onChange={onSortBy} value={`${orderby}-${order}`} />
           </Row>
         )}
-        {isLoading && (
-          <Loading>
-            <Spinner>
-              <CircularProgress size={24} />
-            </Spinner>
-          </Loading>
+        {!loaded && isLoading && (
+          <LoadingContainer>
+            <LoadingSpinner animationSize={50} circleSize={6} />
```

Comment:
50 is quite large. Let's go with half of this for now, 25.

Here and also in `InfiniteScroller`. We can pass `animationSize` as a prop there.

Reply:
<img width="338" alt="Screenshot 2022-06-14 at 10 22 16" src="https://user-images.githubusercontent.com/237508/173543411-0e8fc11c-c428-4377-81cd-a07a7c03675c.png">

We have bigger elsewhere.

Reply:
Hmm but not the same size 50 either. I'll create a separate ticket to unify these then. Easiest.

Reply:
I added the props in. I had to change both size and circles. So I pass an object instead. Seems to work well.